test/commands/encrypt.js

```diff
@@ -76,35 +85,186 @@ describe('lisky encrypt command palette', () => {
 			encryptStub.restore();
 		});
 
-		it('should handle valid parameters', () => {
-			return vorpal.exec(command)
-				.then(() => {
-					(encryptStub.calledWithExactly(message, secret, recipient))
-						.should.be.true();
+		describe('with prompt for password', () => {
+			const isTTY = process.stdin.isTTY;
```

Comment:
this variable name could be more precise that it is the default or initial value for process.stdin.isTTY